modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/LiferayExperienceCloud/components/SetupLXCModal/components/SetupLXCForm/components/SetupLXCPage/hooks/useSubmitLxcEnvironment.js

```diff
@@ -0,0 +1,122 @@
+/**
```

Comment:
@claricebrasil, rename the file to useSubmitLXCEnvironment

Reply:
Ok, done!